app/extensions/edm/__init__.py

```diff
@@ -138,6 +138,38 @@ def initialize_edm_admin_user(self, email, password):
             )
             return False
 
+    # The edm API returns a success and a result, this processes it to just return the result to the caller
+    # TODO is this identical for acm and therefore, should this move into the RestManager
+    def request_passthrough_result(
```

Comment:
thanks for doing this. am torn about unwinding the places where i have passed edm results through _as-is_ to houston endpoints.   i guess we should, for consistency?   would like to talk to jp about exactly why he established this pattern for wbia -- just to see if i will have any regrets.   but then it would take coordinating with ben on where those changes would be.   going to put this on back-burner; but this method will come in handy.   i digress.

Reply:
Yeah that change was simple code de-duplication :)